schema.sql

```diff
@@ -1,128 +1,156 @@
-INSERT INTO Role (Name, DepartmentID, BandID)
+INSERT INTO role (name, department_id, band_id)
 VALUES ('Junior Business Development Manager', 1, 5), ('Junior Account Manager', 1, 5), ('Sales Executive', 1, 5), 
 ('Territory Manager', 1, 5), ('Senior Presales Consultant', 1, 5), ('Senior Marketing Executive', 1, 5);
-insert into Role (Name, DepartmentID, BandID)
+insert into role (name, department_id, band_id)
 VALUES ('Sales Associate', 1, 6), ('Senior Sales Development Representative', 1, 6), ('Presales Consultant', 1, 6), 
 ('Marketing Executive', 1, 6);
-INSERT INTO Role (Name, DepartmentID, BandID)
+INSERT INTO role (name, department_id, band_id)
 VALUES ('Sales Associate', 1, 7), ('Sales Development Representative', 1, 7), ('Presales Associate', 1, 7), 
 ('Marketing Associate', 1, 7);
-INSERT INTO Role (Name, DepartmentID, BandID)
+INSERT INTO role (name, department_id, band_id)
 VALUES ('Sales Development Representative', 1, 8), ('Trainee Presales Associate', 1, 8), ('Marketing Assistant', 1, 8);
 
-INSERT INTO Role_Capability (RoleID, CapabilityID)
+INSERT INTO role_capability (role_id, capability_id)
 VALUES (1,1), (1,2), (2,3), (3,4), (3,5), (4,6);
-INSERT INTO Role_Capability (RoleID, CapabilityID)
+INSERT INTO role_capability (role_id, capability_id)
 VALUES (5,1), (5,2), (6,3), (7,4), (7,5), (8,6);
-INSERT INTO Role_Capability (RoleID, CapabilityID)
+INSERT INTO role_capability (role_id, capability_id)
 VALUES (9,1), (10,2), (11,3), (12,4), (13,5), (14,6);
-INSERT INTO Role_Capability (RoleID, CapabilityID)
+INSERT INTO role_capability (role_id, capability_id)
 VALUES (15,1), (16,2), (17,3), (18,4), (19,5), (20,6);
-INSERT INTO Role_Capability (RoleID, CapabilityID)
+INSERT INTO role_capability (role_id, capability_id)
 VALUES (21,1), (22,2), (23,3), (24,4), (25,5), (26,6);
-INSERT INTO Role_Capability (RoleID, CapabilityID)
+INSERT INTO role_capability (role_id, capability_id)
 VALUES (27,3), (28,4), (29,5), (30,6);
-INSERT INTO Role_Capability (RoleID, CapabilityID)
+INSERT INTO role_capability (role_id, capability_id)
 VALUES (31,3), (32,4), (33,5), (34,6);
-INSERT INTO Role_Capability (RoleID, CapabilityID)
+INSERT INTO role_capability (role_id, capability_id)
 VALUES (35,4), (36,5), (37,6);
+
+-- ---------------------------------- VIEWS ------------------------------- --
+
+CREATE VIEW viewDepartments AS 
+SELECT department_id, Name
+FROM department;
+
+/** 
+CREATE VIEW viewRoles AS
+SELECT Department.DepartmentID, Department.Name AS 'Department Name', 
+Capability.CapabilityID, Capability.Name AS 'Capability Name', Band.BandID, Band.Name AS 'Band Name',
+Role.RoleID, Role.Name AS 'Role Name'
+FROM Department JOIN Capability ON Department.DepartmentID = Capability.DepartmentID JOIN Role_Capability ON
+Role_Capability.CapabilityID = Capability.CapabilityID JOIN Role ON Role_Capability.RoleID = Role.RoleID 
+JOIN Band ON Role.BandID = Band.BandID
+WHERE Department.Name = 'Sales & Marketing';
```

Comment:
Having commented out code will be confusing to anyone pulling from master, if this code isn't needed, can it be removed?